x/clock/abci.go

```diff
@@ -0,0 +1,42 @@
+package clock
+
+import (
+	"log"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/CosmosContracts/juno/v17/x/clock/keeper"
+	"github.com/CosmosContracts/juno/v17/x/clock/types"
+)
+
+// EndBlocker executes on contracts at the end of the block.
+func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
+	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyEndBlocker)
+
+	message := []byte(types.EndBlockSudoMessage)
+
+	p := k.GetParams(ctx)
+
+	errorExecs := make([]string, len(p.ContractAddresses))
```

Comment:
Contracts should really never error unless they either 
a) Do not exist yet (useful for testing
b) Migrate and remove / alter the original SudoMsg they had before original upload.